### PR TITLE
OSDOCS-10781: Style fix: Add {nbsp}

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-10781](https://issues.redhat.com//browse/OSDOCS-10781): Style fix: Add {nbsp} between "Red" and "Hat"

Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-10781

Link to docs preview:
Too many previews to link.

QE review:
No QE needed.

Additional information:

### CODE REVIEW

File: modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ You can use an AWS-managed policy or a customer-managed policy to set the bounda
 
 [NOTE]
 ====
-This feature is only supported on Red Hat OpenShift Service on AWS (classic architecture) clusters.
+This feature is only supported on Red{nbsp}Hat OpenShift Service on AWS (classic architecture) clusters.
```
Comment (EricPonvelle) [inline modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc:14]: This module isn't used in any assemblies.

File: modules/rosa-sts-byo-oidc.adoc [hunk 2]
```diff
@@ -48,7 +48,7 @@ This command returns the following information.
 +
 [source,terminal]
 ----
-? Would you like to create a Managed (Red Hat hosted) OIDC Configuration Yes
+? Would you like to create a Managed (Red{nbsp}Hat hosted) OIDC Configuration Yes
```
Comment (EricPonvelle) [inline modules/rosa-sts-byo-oidc.adoc:51]: image

File: modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc [hunk 3]
```diff
@@ -38,7 +38,7 @@ endif::[]
 +
 [NOTE]
 ====
-If your AWS account ID is not listed, check that you have successfully associated your AWS account with your Red Hat organization. If your account role ARNs are not listed, check that the required account-wide STS roles exist in your AWS account.
+If your AWS account ID is not listed, check that you have successfully associated your AWS account with your Red{nbsp}Hat organization. If your account role ARNs are not listed, check that the required account-wide STS roles exist in your AWS account.
 ====
 
 . Click *Next*.
```
Comment (EricPonvelle) [inline modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc:44]: Can Can you fix the `====` on line 48? It should be at line 49 with the note at 48.

image

File: modules/rosa-sts-setting-up-environment.adoc [hunk 4]
```diff
@@ -77,7 +77,7 @@ $ rosa
 .Example output
 [source,terminal]
 ----
-Command line tool for Red Hat OpenShift Service on AWS.
+Command line tool for Red{nbsp}Hat OpenShift Service on AWS.
```
Comment (EricPonvelle) [inline modules/rosa-sts-setting-up-environment.adoc:80]: image

File: modules/rosa-sts-setting-up-environment.adoc [hunk 5]
```diff
@@ -142,7 +142,7 @@ $ rosa login
 .Example output
 [source,terminal]
 ----
-To login to your Red Hat account, get an offline access token at https://console.redhat.com/openshift/token/rosa
+To login to your Red{nbsp}Hat account, get an offline access token at https://console.redhat.com/openshift/token/rosa
```
Comment (EricPonvelle) [inline modules/rosa-sts-setting-up-environment.adoc:145]: image

File: modules/rosa-sts-setting-up-environment.adoc [hunk 6]
```diff
@@ -196,7 +196,7 @@ OCM Account Name:             Your Name
 OCM Account Username:         you@domain.com
 OCM Account Email:            you@domain.com
 OCM Organization ID:          1HopHfA2hcmhup5gCr2uH5aaaaa
-OCM Organization Name:        Red Hat
+OCM Organization Name:        Red{nbsp}Hat
```
Comment (EricPonvelle) [inline modules/rosa-sts-setting-up-environment.adoc:199]: image

**Review by EricPonvelle — COMMENTED** [review]
I think there were only a few items that actually needed to be addressed before we can merge this ticket. Anything out of scope can be addressed in another PR.

File: welcome/cloud-experts-rosa-hcp-sts-explained.adoc [hunk 7]
```diff
@@ -15,7 +15,7 @@ toc::[]
 
 [id="credential-methods-rosa-hcp"]
 == AWS STS credential method
-As part of {hcp-title}, Red Hat must be granted the necessary permissions to manage infrastructure resources in your AWS account.
+As part of {hcp-title}, Red{nbsp}Hat must be granted the necessary permissions to manage infrastructure resources in your AWS account.
```
[the remark this answers is not in the document]
Comment (jneczypor) [reply welcome/cloud-experts-rosa-hcp-sts-explained.adoc:18]: The variable is `:hcp-title-first: {product-title} (ROSA) with {hcp} (HCP)` so I'm leaving it as-is for now

File: rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-private-cluster.adoc [hunk 8]
```diff
@@ -14,7 +14,7 @@ Privacy settings can be configured during cluster creation or after a cluster is
 ////
 [NOTE]
```
[the remark this answers is not in the document]
Comment (jneczypor) [reply rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-private-cluster.adoc:15]: Save for future PR

File: modules/creating-a-machine-pool-ocm.adoc [hunk 9]
```diff
@@ -30,7 +30,7 @@ endif::openshift-rosa[]
 ifndef::openshift-rosa[]
```
[the remark this answers is not in the document]
Comment (jneczypor) [reply modules/creating-a-machine-pool-ocm.adoc:30]: Saving for a later PR.

File: modules/osd-aws-privatelink-about.adoc [hunk 10]
```diff
@@ -16,7 +16,7 @@ ifdef::rosa-hcp[]
 All {hcp-title} clusters are created with an AWS PrivateLink connection to expose the private Kubernetes API server to the customer's virtual private cloud (VPC).
 endif::rosa-hcp[]
 ifndef::rosa-hcp[]
-A {product-title} cluster can be created without any requirements on public subnets, internet gateways, or network address translation (NAT) gateways. In this configuration, Red Hat uses AWS PrivateLink to manage and monitor a cluster to avoid all public ingress network traffic. Without a public subnet, it is not possible to configure an application router as public. Configuring private application routers is the only option.
+A {product-title} cluster can be created without any requirements on public subnets, internet gateways, or network address translation (NAT) gateways. In this configuration, Red{nbsp}Hat uses AWS PrivateLink to manage and monitor a cluster to avoid all public ingress network traffic. Without a public subnet, it is not possible to configure an application router as public. Configuring private application routers is the only option.
```
[the remark this answers is not in the document]
Comment (jneczypor) [reply modules/osd-aws-privatelink-about.adoc:19]: Saving for a later PR

File: modules/rosa-unlinking-and-deleting-ocm-and-user-iam-roles.adoc [hunk 11]
```diff
@@ -18,7 +18,7 @@ endif::hcp[]
 ifdef::hcp[]
 {hcp-title} 
 endif::hcp[]
-cluster by using {cluster-manager-first}, you also create {cluster-manager} and user Identity and Access Management (IAM) roles that link to your Red Hat organization. After deleting your cluster, you can unlink and delete the roles by using the ROSA CLI (`rosa`).
+cluster by using {cluster-manager-first}, you also create {cluster-manager} and user Identity and Access Management (IAM) roles that link to your Red{nbsp}Hat organization. After deleting your cluster, you can unlink and delete the roles by using the ROSA CLI (`rosa`).
```
[the remark this answers is not in the document]
Comment (jneczypor) [reply modules/rosa-unlinking-and-deleting-ocm-and-user-iam-roles.adoc:21]: Saving for a later PR

File: modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc (hunk 1 above)
Comment (jneczypor) [reply modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc:14]: Poor guy. He is lonely.

File: modules/rosa-adding-taints.adoc [hunk 12]
```diff
@@ -19,7 +19,7 @@ A cluster must have at least one machine pool that does not contain any taints.
 ifndef::openshift-rosa[]
 .Prerequisites
 // ifdef::openshift-rosa[]
```
[the remark this answers is not in the document]
Comment (jneczypor) [reply modules/rosa-adding-taints.adoc:21]: Saving for a later PR.

File: rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc [hunk 13]
```diff
@@ -8,7 +8,7 @@ toc::[]
 ifdef::openshift-dedicated[]
 [IMPORTANT]
 ====
-Autoscaling is available only on clusters that were purchased through the Red Hat Marketplace.
+Autoscaling is available only on clusters that were purchased through the Red{nbsp}Hat Marketplace.
```
[the remark this answers is not in the document]
Comment (jneczypor) [reply rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc:11]: Saving for a later PR.

**Pull request merged**